src/authentication/authentication-hapi-plugin.ts

```diff
@@ -0,0 +1,154 @@
+
```

Comment:
Some files have empty first lines, some don't. It'd be good to make this consistent. In UI projects, we've started making files not have empty lines at the beginning.